src/main/resources/swagger/api-docs.yaml

```diff
@@ -8138,6 +8138,12 @@ components:
         allUsersGroup:
           type: boolean
           description: User is a member of the "All Users" group?
+        adminEnabled:
+          type: boolean
+          description: User is a member of the admin group?
```

Comment:
I was confused by the naming of this `adminEnabled` field when I first saw it, but it's actually to indicate whether or not a Terra admin has explicitly disabled a user in the system. So maybe a more fitting description would be "User is enabled in the system?" or something similar

Reply:
thanks, I was just guessing.  Updated.

Reply:
... but somehow it got lost in the squash/merge.  Trying again: #926